code/modules/projectiles/projectile.dm

```diff
@@ -73,21 +73,20 @@
 
 /obj/item/projectile/is_hot()
 	if (damage_type == BURN)
-		return damage*heat
+		return damage * heat
 
-/obj/item/projectile/multiply_projectile_damage(var/newmult)
-	damage = initial(damage)*newmult
+/obj/item/projectile/multiply_projectile_damage(newmult)
+	damage = initial(damage) * newmult
 
-/obj/item/projectile/proc/on_hit(var/atom/target, var/blocked = 0, var/def_zone = null)
+/obj/item/projectile/proc/on_hit(atom/target, def_zone = null)
 	if(!isliving(target))	return 0
 	if(isanimal(target))	return 0
-	if(blocked >= 2)		return 0//Full block
```

Comment:
This part is important since it gets called by things like shields to block it.

Reply:
Called where? I can find any calls for it, including in shields.

Reply:
Just realized you meant riot shields and such.